tests/appsec/rasp/__init__.py

```diff
@@ -0,0 +1,42 @@
+def validate_rasp_attack(span, rule, parameters=None):
```

Comment:
Should we check the presence of a stack_id?

Reply:
I want to add a separate check to validate the presence of `stack_id` with the contents of the stack trace, so I skipped it for now as it's very hard to verify without a library.